microservice-transaction-sample/README.md

```diff
@@ -1,225 +1,183 @@
 
-## Setup
+```console
+$ cd scalardb-samples/microservice-transaction-sample
+```
 
 ### Start Cassandra and MySQL
 
-To start Cassandra and MySQL, you need to run the following `docker-compose` command:
+Cassandra and MySQL are already configured for the sample application, as shown in [database.properties](database.properties). For details about configuring the multi-storage transactions feature in ScalarDB, see [How to configure ScalarDB to support multi-storage transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md#how-to-configure-scalardb-to-support-multi-storage-transactions).
 
-```shell
-$ docker-compose up -d cassandra mysql
+To start Cassandra and MySQL, which are included in the Docker container for the sample application, run the following command:
+
+```console
+$ docker-compose up -d
 ```
 
-Please note that you need to wait around more than one minute for the containers to be fully started.
+{% capture notice--info %}
+**Note**
+
+Starting the Docker container may take more than one minute depending on your development environment.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-### Load schema
+### Load the schema
 
-You then need to apply the schema with the following commands.
-To download the schema loader tool, `scalardb-schema-loader-<VERSION>.jar`, see the [Releases](https://github.com/scalar-labs/scalardb/releases) of ScalarDB and download the version that you want to use.
+The database schema (the method in which the data will be organized) for the sample application has already been defined in [customer-service-schema.json](customer-service-schema.json) for the Customer Service and [order-service-schema.json](order-service-schema.json) for the Order Service.
 
-For MySQL:
+To apply the schema, go to the [ScalarDB Releases](https://github.com/scalar-labs/scalardb/releases) page and download the ScalarDB Schema Loader that matches the version of ScalarDB that you want to use to the `scalardb-samples/microservice-transaction-sample` folder.
 
-```shell
+#### MySQL
+
+To load the schema for [customer-service-schema.json](customer-service-schema.json) into MySQL, run the following command, replacing `<VERSION>` with the version of the ScalarDB Schema Loader that you downloaded:
+
+```console
 $ java -jar scalardb-schema-loader-<VERSION>.jar --config database-mysql.properties --schema-file customer-service-schema.json
 ```
 
-For Cassandra:
+#### Cassandra
+
+To load the schema for [order-service-schema.json](order-service-schema.json) into Cassandra, run the following command, replacing `<VERSION>` with the version of the ScalarDB Schema Loader that you downloaded:
 
-```shell
+```console
 $ java -jar scalardb-schema-loader-<VERSION>.jar --config database-cassandra.properties --schema-file order-service-schema.json --coordinator
 ```
 
-### Start Microservices
+#### Schema reference
```

Comment:
```suggestion
#### Schema details
```
What do you mean by `reference`?
I think it explains more details about the database schema, so `schema reference` sounds a bit weird to me.

Reply:
I'm OK with `Schema details` here. Both imply that this section is supplemental information, but I can see how the word `reference` could imply that the schema is referencing something else.